### DISCUSSION TITLE
New business owner

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#395] [General]
Hey guys. I am a designer and was working in fashion industry for more than 10 years. I am finally ready to start my own business and would love to hear some advices . Thanks in advance

**Comment by hernessteve** [comment]
My only advice - never give up!

**Comment by XelaAl** [comment]
Good for you! I think you need an [omnichannel retail](https://botsify.com/blog/optimize-omnichannel-retail-strategy-fashion-industry/) strategy. Thus, you will know how to sell your products and attract customers. And it has to be noted, that both an online and offline market are important. I'd start by creating a user-friendly platform for selling purposes.
